README.md

```
@@ -1,207 +1,168 @@
 
----
+![Final](Add photo of final product here) Explain the final build
 
 ## Additional Documentation
 
-### For Web Projects with Backend:
+For Web Projects with Backend:
 
-#### API Documentation
+API Documentation
+Base URL: https://api.yourproject.com
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)